dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi12/fat/tests/BeanDiscoveryModeNoneTest.java

```diff
@@ -26,28 +27,34 @@
 import org.jboss.shrinkwrap.api.spec.ResourceAdapterArchive;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 
-import com.ibm.ws.fat.util.BuildShrinkWrap;
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
 import com.ibm.ws.fat.util.LoggingTest;
-import com.ibm.ws.fat.util.ShrinkWrapSharedServer;
+import com.ibm.ws.fat.util.SharedServer;
 import com.ibm.ws.fat.util.browser.WebBrowser;
+import com.ibm.ws.fat.util.browser.WebBrowserException;
+import com.ibm.ws.fat.util.browser.WebBrowserFactory;
+import com.ibm.ws.fat.util.browser.WebResponse;
 
 import componenttest.annotation.ExpectedFFDC;
 import componenttest.custom.junit.runner.Mode;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
 
 public class BeanDiscoveryModeNoneTest extends LoggingTest {
 
-    @ClassRule
-    // Create the server.
-    public static ShrinkWrapSharedServer SHARED_SERVER = new ShrinkWrapSharedServer("cdi12BeanDiscoveryModeNoneServer");
+    private static LibertyServer server;
 
-    @BuildShrinkWrap
-    public static Archive buildShrinkWrap() {
-         
-         return ShrinkWrap.create(WebArchive.class, "beanDiscoveryModeNone.war")
+    @BeforeClass
+    public static void setUp() throws Exception {
+         server = LibertyServerFactory.getLibertyServer("cdi12BeanDiscoveryModeNoneServer"); 
```

Comment:
You can take this line out and add `@Server` to the `LibertyServer` field instead.